docs/dev-guide.md

```diff
@@ -149,9 +149,9 @@ var ERR = require('async-stacktrace');
 var _ = require('lodash');
 var express = require('express');
 var router = express.Router();
-var sqldb = require('../prairielib/sql-db'); // adjust path as needed
+var sqldb = require('@prairielearn/postgres') // adjust path as needed
```

Comment:
I'm wondering if we should significantly trim back the dev guide? A lot of the advice is outdated, incorrect, or doesn't reflect the current patterns we're following.

Reply:
I'm happy to have it significantly trimmed back. Most of the code-style stuff is horribly outdated at this point.